tasks/cleanup-vm/examples/secrets/ssh-secret.yaml

```
@@ -8,5 +8,4 @@ stringData:
   disable-strict-host-key-checking: "true"
   ssh-privatekey: |
```

```suggestion
  ssh-privatekey: |-
```

Could possible help to get rid of whitespace at the end.